frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard-footer.tsx

```diff
@@ -65,13 +65,12 @@ const CreateVMWizardFooterComponent: React.FC<CreateVMWizardFooterComponentProps
   isSimpleView,
   goToStep,
   onEdit,
-  setActiveNS,
   isInvalidUserTemplate,
 }) => {
   const { t } = useTranslation();
   const [showError, setShowError, checkValidity] = useShowErrorToggler();
+  const [, setActiveNS] = useLastNamespace();
   const activeNS = getActiveNamespace();
```

Comment:
Why not use the value from the context here?

Reply:
missed this one!

Reply:
fixed!